python/ray/exceptions.py

```diff
@@ -326,6 +331,12 @@ def from_task_error(task_error: RayTaskError):
         return RayActorError(task_error)
 
 
+class RayPreemptionError(RayActorError):
```

Comment:
So we don't support task?

Reply:
we are going to merge the minimal needed now for this PR (and follow up will contain other cases).

NOTE: Currently, task error architecture is not that great, and we cannot really detect correct error